src/api/entities/Identity/index.ts

```diff
@@ -255,4 +259,23 @@ export class Identity extends Entity<UniqueIdentifiers> {
 
     return checkedRoles.every(hasRole => hasRole);
   }
+
+  /**
+   * Get the list of tokens for which this identity is a trusted claim issuer
+   */
+  public async getTrustingTokens(
+    args: { order: Order } = { order: Order.Asc }
+  ): Promise<SecurityToken[]> {
+    const { context, did } = this;
+
+    const { order } = args;
+
+    const {
+      data: { tokensByTrustedClaimIssuer: tickers },
+    } = await context.queryMiddleware<Ensured<Query, 'tokensByTrustedClaimIssuer'>>(
+      tokensByTrustedClaimIssuer({ claimIssuerDid: did, order })
+    );
+
+    return tickers.map(ticker => new SecurityToken({ ticker }, context));
```

Comment:
Tickers from harvester comes with `\u0000` to complete 12 characters. You should trim them off to return tickers.